packages/patterns-react/src/patterns/CardSection/CardSection.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import root from 'window-or-global';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import classNames from 'classnames';
+import { featureFlag, sameheight } from '@carbon/ibmdotcom-utilities';
+import { DDS_CARD_SECTION } from '../../internal/FeatureFlags';
+import { CardLink } from '@carbon/ibmdotcom-react';
+import { ArrowRight20 } from '@carbon/icons-react';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Card section Component
+ *
+ * @param {object} props props object
+ * @param {string} props.title cards group title
+ * @param {string} props.theme theme name
+ * @param {Array} props.cards Array of object with title, href and target properties
+ * @returns {object} JSX Object
+ */
+const CardSection = ({ title, cards, theme }) => {
+  const containerRef = useRef();
+
+  useEffect(() => {
+    root.addEventListener('resize', sameheight(containerRef.current, 'md'));
```

Comment:
Here's what I'm seeing in ways of `sameheight`. Everything should align.

![image](https://user-images.githubusercontent.com/3793636/71923688-3174b480-3153-11ea-8156-988987d5db55.png)

It also looks like you're not calling it on `onload` unless I missed this too somewhere else. That's why I didn't see anything. On resize I'm sttill not even seeing any changes to the HTML to try and set them all to the sameheight.

Reply:
I see, let me look into this.

Reply:
I am calling the `sameheight` function inside the `useEffect` hook

Reply:
@photodow: made some changes and it is working fine but the `sameheight` helper function has some bugs, had a short call with @RaphaelAmadeu  he is making changes in utility function.